Testing/SystemTests/tests/framework/D7AbsoluteCrossSectionsTest.py

```diff
@@ -0,0 +1,245 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import systemtesting
+from mantid.simpleapi import *
+
+
+class ILL_D7_Powder_Test(systemtesting.MantidSystemTest):
+
+    def __init__(self):
+        super(ILL_D7_Powder_Test, self).__init__()
+        self.setUp()
```

Comment:
Btw, both of these must be classmethods, `setUpClass` and `tearDownClass`, so that they are run before and after the whole suite and not before and after each individual test.

Reply:
Good point, I've edited my example :smile:

Reply:
Yes, absolutely, I am going to change it according to your suggestion. I reused the template from tests familiar to me, thus the structure. We will need to look into those to do some maintenance at some point.

Reply:
Yes, 5 years ago I didn't know about this, but we should not repeat the same mistakes.

Reply:
>     * This can be completely removed, setUp() is always called before the tests run by systemTesting and this in effect runs the setup twice: https://github.com/mantidproject/mantid/blob/master/Testing/SystemTests/lib/systemtests/systemtesting.py#L244
> 
>     * Secondly we need a `tearDown()` that restores everything back, as some tests rely on the config being in a known starting state (incorrectly) and will turn flaky without care
> 
> 
> ```
> @classmethod
> def setUpClass(cls):
>    cls._original_facility = config['default.facility']
>    cls._original_instrument = config['default.instrument']
>    cls._data_search_dirs = config.getDataSearchDirs()
>    # ... config setting
> 
> @classmethod
> def tearDownClass(cls):
>    config['default.facility'] = cls._original_facility
>    config['default.instrument'] = cls._original_instrument
>    config.setDataSearchDirs() = cls._data_search_dirs
> ```

Just a small change there, the last line should be `config.setDataSearchDirs(cls._data_search_dirs)`.